source/slang/slang-check-impl.h

```diff
@@ -950,20 +956,48 @@ namespace Slang
             DeclRef<InterfaceDecl>  interfaceDeclRef,
             DeclRef<Decl>           requirementDeclRef);
 
-        bool doesTypeConformToInterfaceImpl(
-            RefPtr<Type>            originalType,
-            RefPtr<Type>            type,
-            DeclRef<InterfaceDecl>  interfaceDeclRef,
+            /// Check whether `subType` is declared a sub-type of `superTypeDeclRef`
```

Comment:
Should indentation of these comments be aligned with function signature?

Reply:
This is something you would need to discuss with @jsmall-nvidia. He has prior experience with a codebase that uses the convention you see here (documentation comments are indented one level relative to the thing they document), and prefers that style. I have no strong preference, and have defaulted to using that style to keep new code internally consistent.

Reply:
I see. I am fine wit that. Maybe we could set up a clang-format file and do a formatting pass some time to get all these formatting consistent. But if the tools doesn't support this style I suggest we move to something that are better supported by clang-format.

Reply:
I agree that if we set up something automated (which would almost certainly be clang-format), we should be willing to change our conventions to something that can be enforced automatically.

Setting up clang-format has been one of those long-standing "would be nice" things that nobody has had enough time to sink into just yet. If you are volunteering, though... :)